server.rb

```diff
@@ -0,0 +1,44 @@
+require 'sinatra'
+require 'pry'
+require 'csv'
+
+configure do
+  enable :sessions
+end
+
+articles_array = []
+
+get '/articles' do
+  # binding.pry
+  articles_array = CSV.read("./articles.csv")
```

Comment:
you might want to refactor this to be a method call to not over load that path and allow it to be reused in multiple place with out duplicating your code. DRY it up as they say

```ruby
def load_articlesv(path = "./articles.csv")
   CSV.read(path)
end
```

then you can do 
```ruby
articles_array = load_articles
```

or per the earlier comment on the html you could do something like
```ruby
def parsed_articles
  raw_articles = load_articles
  articles = []
  raw_articles.each do |raw|
    article = Hash.new
    article[:url] = raw[0]
    article[:description] = raw[1]
    articles << article
  end
  return articles
end
```